docs-kits/kits/behaviour-twin-kit/adoption-view/guidance-and-resources.md

```diff
@@ -0,0 +1,37 @@
+﻿---
```

Comment:
Does it make sense to also move it rather to the development view?

Reply:
Will be done.

Reply:
Done.